### PR TITLE
VPN-3680: Update controller view strings

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
## Description

- Adds new accessible label strings for the speed test and connection information/ip panel buttons
- Removes old strings

## Reference

VPN-3680

## Checklist
    
- [ ] My code follows the style guidelines for this project
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: translations/strings.yaml [hunk 1]
```diff
@@ -370,6 +369,9 @@ genericError:
 # Connection Info Screen
 connectionInfo:
   openButton: Connection Info
+  startSpeedTest:
+    value: Start speed test
+    comment: Accessible label for the button that opens the speed test view and initiates a speed test
```
Comment (flodolo) [inline translations/strings.yaml:374]: Accessible or Accessibility?

**Review by flodolo — APPROVED** [review]

File: translations/strings.yaml [hunk 2]
```diff
@@ -340,7 +340,6 @@ dnsOverwriteDialog:
   primaryButton: Confirm
   secondaryButton: Maybe later
 
-aboutUs:
```
Comment (MattLichtenstein) [inline translations/strings.yaml:343]: probably need to add this back

**Review by MattLichtenstein — CHANGES_REQUESTED** [review]
Also, should we remove the old strings that are being replaced?

**Comment by flodolo** [comment]
> Leaves old strings put for now and can be deleted when no longer needed in release branch

This is not really necessary, automation takes care of keeping strings needed by release and release branches.

File: translations/strings.yaml (hunk 2 above)
Comment (lesleyjanenorton) [reply translations/strings.yaml]: done, thanks!

**Review by MattLichtenstein — APPROVED** [review]

**Pull request merged**